components/CodeBlock.tsx

```diff
@@ -1,39 +1,84 @@
+import * as Components from '@atom-learning/components'
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import theme from 'prism-react-renderer/themes/nightOwl'
 import * as React from 'react'
+import { LiveEditor, LiveError, LivePreview, LiveProvider } from 'react-live'
 
 import { styled } from '../stitches.config'
 
+type CodeBlockProps = {
+  children: string
+  live?: boolean
+  className?: string
+}
+
 const StyledPre = styled('pre', {
   borderRadius: '$1',
   display: 'block',
   fontSize: '15px',
-  lineHeight: 1.5,
   fontWeight: 100,
+  lineHeight: 1.5,
+  mb: '$4',
+  mt: 0,
   mx: '-$4',
```

Comment:
I just so I learn, is the `-` part of stitches?

Reply:
`-` is changing the token value to a negative, so `$4` is `32px`, `-$4` is `-32px`.